packages/hardware-trezor/src/TrezorKeyAgent.ts

```diff
@@ -151,10 +34,6 @@ export class TrezorKeyAgent extends KeyAgentBase {
       }
 
       const signedData = result.payload;
-      if (signedData.hash !== tx.hash) {
-        throw new errors.HwMappingError('Trezor computed a different transaction id');
-      }
```

Comment:
Just saw this comment https://github.com/input-output-hk/cardano-js-sdk/pull/734#discussion_r1213916043 - did you remove this on purpose?

Reply:
@marcin-mazurek yes, because if you add it everything is broken, but this is not "our" fault in this PR (the existing Trezor implementation is broken)

Reply:
Understood, as discussed on Slack - let's raise a separate task for it

Reply:
Consider skipping the tests instead (keeping the tx hash check)